iree/compiler/Dialect/LinalgExt/IR/LinalgExtOps.td

```diff
@@ -31,10 +34,28 @@ class LinalgExt_Op<string mnemonic, list<OpTrait> traits = []> :
 // Non-structured ops
 //===----------------------------------------------------------------------===//
 
-def LinalgExt_SortOp : LinalgExt_Op<"sort", [
-    AttrSizedOperandSegments,
-    DeclareOpInterfaceMethods<MemoryEffectsOpInterface>,
-    SingleBlockImplicitTerminator<"YieldOp">]> {
+def LinalgExt_ScatterOp : LinalgExt_Op<"scatter"> {
+  let summary = "Scatter operator";
+  let description = [{
+    Based on XLA operation semantics, generates a result which is the
+    value of the input array `operand`, with several slices (at
+    indices specified `indices`) updated with the values in `updates`
+    using `update_computation`.
```

Comment:
Can we add a description for region arguments? I think we can say that it must contain to block argument. The first argument is update value and the second argument is init value?

mhlo.scatter has a reverse form -- `output[O] = update_computation(output[O], updates[U])`.

I think `output[O] = update_computation(updates[U], output[O])` matches Linalg ops more. We can handle the mapping when lowering mhlo.scatter to linalg_ext ops.

Reply:
Ok, I updated the description